### ISSUE TITLE
【打字机】拉丁粤文标调符号位置不正确

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lotem/zime#28]
```
需要改的地方：
1) iu  和 ui  
韵里，因为前字母是主韵腹，后字母只是韵尾，所以调号应��
�加在前字母。现在该输入法却加在了后字母。
2) eo, eu, ea 韵和韵腹里，调号应放在非e的 o u 或 a 
上。但现在该输入法却加在了e上。 

仍应该保持在现有状况的：
1) ua, uo, ue 韵或韵腹里, 不应该加在 u 
上，现在该输入法加在非 u 上的做法仍是正确的。所以要和 
ui 刚好相反。要为 ui 特别开小灶。
2）ia, io, ie, ei,  
因为粤语没有该韵腹或韵，所以加在哪都可以。所以不需要��
�心现在的设置。不过我想，不加在 i 
上应该是常理。因为广东次方言片里应该有 iau 
这样的韵。在这种韵里，i 是介音，u 是韵尾，a 
是韵腹。所以调号加在 a 上是常理。也就是说 i 和 u 
作为介音的特性是一致的。可以绝对把 i u 
同归在「小灶一类」。

What version of the product are you using? On what operating system?
Typewriter 0.7.1 （zime-web线上服务）

Please provide any additional information below.


```

Original issue reported on code.google.com by `chen....@gmail.com` on 22 Aug 2010 at 5:29

**Comment by GoogleCodeExporter** [comment]
```
受一段与ZIME中拼写运算相仿的js脚本控制
```

Original comment by `chen....@gmail.com` on 22 Aug 2010 at 5:33

**Comment by GoogleCodeExporter** [comment]
Original comment by `chen....@gmail.com` on 28 Aug 2010 at 9:04
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
請問能否 製成 能打大寫字母的，多謝。

```

Original comment by `penky...@gmail.com` on 28 Aug 2010 at 11:31

**Comment by GoogleCodeExporter** [comment]
```
彿振兄：

Typewriter 7.2 仍有少許bug.

I agree with Penkyamp

ei e
ia a
io o
ie e

ia, io, ie, ei,  
因为粤语没有该韵腹或韵，所以加在哪都可以。所以不需要��
�心现在的设置。不过我想，不加在 i 
上应该是常理。因为广东次方言片里应该有 iau 
这样的韵。在这种韵里，i 是介音，u 是韵尾，a 
是韵腹。所以调号加在 a 上是常理。也就是说 i 和 u 
作为介音的特性是一致的。可以绝对把 i u 
同归在「小灶一类」。

jië jiä jiö jeï

ei 仍有問題
請加在 e 的主韻母上。
```

Original comment by `titus...@gmail.com` on 29 Aug 2010 at 10:55

**Comment by GoogleCodeExporter** [comment]
```
"ei" fixed in typewriter-0.7.3
```

Original comment by `chen....@gmail.com` on 1 Sep 2010 at 4:31
